packages/remix/src/generators/convert-to-inferred/lib/build-post-target-transformer.ts

```diff
@@ -0,0 +1,236 @@
+        tree,
+        target.options,
+        projectDetails.projectName,
+        projectDetails.root,
+        configValues['default'],
+        migrationLogs
+      );
+    }
+
+    if (target.configurations) {
+      for (const configurationName in target.configurations) {
+        const configuration = target.configurations[configurationName];
+        configValues[configurationName] ??= {};
+
+        handlePropertiesFromTargetOptions(
+          tree,
+          configuration,
+          projectDetails.projectName,
+          projectDetails.root,
+          configValues[configurationName],
+          migrationLogs
+        );
+      }
+
+      if (Object.keys(target.configurations).length === 0) {
+        if ('defaultConfiguration' in target) {
+          delete target.defaultConfiguration;
+        }
+        delete target.configurations;
+      }
+
+      if (
+        'defaultConfiguration' in target &&
+        !target.configurations[target.defaultConfiguration]
+      ) {
+        delete target.defaultConfiguration;
+      }
+    }
+
+    if (target.outputs) {
+      target.outputs = target.outputs.filter(
+        (out) => !out.includes('options.outputPath')
+      );
+      processTargetOutputs(target, [], inferredTargetConfiguration, {
+        projectName: projectDetails.projectName,
+        projectRoot: projectDetails.root,
+      });
+    }
+
+    setServerBuildPathToUseConfigValueForOutput(
+      tree,
+      remixConfigPath,
+      projectDetails.projectName,
+      migrationLogs
+    );
+    setAssetsBuildDirectoryToUseConfigValueForOutput(
+      tree,
+      remixConfigPath,
+      projectDetails.projectName,
+      migrationLogs
+    );
+    addConfigValuesToConfigFile(tree, remixConfigPath, configValues);
+
+    return target;
+  };
+}
+
+function handlePropertiesFromTargetOptions(
+  tree: Tree,
+  options: any,
+  projectName: string,
+  projectRoot: string,
+  configValues: RemixConfigValues,
+  migrationLogs: AggregatedLog
+) {
+  if ('outputPath' in options) {
+    configValues.outputPath = toProjectRelativePath(
+      options.outputPath,
+      projectRoot
+    );
+    delete options.outputPath;
+  }
+
+  if ('includeDevDependenciesInPackageJson' in options) {
+    migrationLogs.addLog({
+      project: projectName,
+      executorName: '@nx/remix:build',
+      log: "Unable to migrate `includeDevDependenciesInPackageJson` to Remix Config. Use the `@nx/dependency-checks` ESLint rule to update your project's package.json.",
+    });
+
+    delete options.includeDevDependenciesInPackageJson;
+  }
+
+  if ('generatePackageJson' in options) {
+    migrationLogs.addLog({
+      project: projectName,
+      executorName: '@nx/remix:build',
+      log: "Unable to migrate `generatePackageJson` to Remix Config. Use the `@nx/dependency-checks` ESLint rule to update your project's package.json.",
+    });
+
+    delete options.generatePackageJson;
+  }
+
+  if ('generateLockfile' in options) {
```

Comment:
The lint rule won't help here since it only updates the source `package.json` file. We should just say this isn't currently supported.

Reply:
Ok no problem